### PR TITLE
bootstrap_in_place: fix number of params when running butane

### PR DESCRIPTION
**Pull request by vrutkovs**
Ensure that retrying_run_container/run_container gets 4 params

### CODE REVIEW

File: src/tests/test_bootstrap_in_place.py [hunk 1]
```diff
@@ -127,13 +127,17 @@ def embed(self, image_name: str, ignition_file: str, embed_image_name: str) -> s
         os.remove(embedded_image) if os.path.exists(embedded_image) else None
 
         flags = shlex.split(f"--privileged --rm -v /dev:/dev -v /run/udev:/run/udev -v {os.getcwd()}:/data -w /data")
+        cmd = (
+            f"iso ignition embed {BUILD_DIR}/{image_name} "
+            "-f --ignition-file /data/{IBIP_DIR}/{ignition_file}"
```
Comment (omertuc) [inline src/tests/test_bootstrap_in_place.py:132]:
```suggestion
            "-f --ignition-file /data/{IBIP_DIR}/{ignition_file} "
```

**Comment by omertuc** [comment]
I don't see any functional difference between what was before and what you did now, what changed?

**Comment by vrutkovs** [comment]
In https://github.com/openshift/release/pull/35225 the code wanted 4 args but we were passing 5 (not sure why):
```
 INFO Single-Node-Ignition-Config created in: build/ibip and build/ibip/auth 
2023-01-19 13:42:24,998  retry.api WARNING    - 139947996673856 - run_container() takes from 2 to 4 positional arguments but 5 were given, retrying in 30 seconds... 	(/root/.pyenv/versions/3.11.0/lib/python3.11/site-packages/retry/api.py:40)
```

**Comment by omertuc** [comment]
The issue is with `inject_bootstrap`, not `embed`. The former clearly passes 5 arguments

**Comment by vrutkovs** [comment]
Oh, that explains it! Updated

**Comment by omertuc** [comment]
You pushed too much?

**Review by eranco74 — COMMENTED** [review]
/lgtm

**Review by osherdp — APPROVED** [review]

**Comment by vrutkovs** [comment]
/hold cancel

Tests are not failing anymore

**Comment by osherdp** [comment]
failures are not relevant to those changes
https://github.com/openshift/release/pull/36881

**Pull request merged**